dev-tools/composer.json

```diff
@@ -1,18 +1,18 @@
 {
     "require": {
-        "php": "^7.3"
+        "php": "~8.0.0"
```

Comment:
why `~` and not `^` ?

Reply:
So that it does not match PHP 8.1+.

Reply:
I understand that part, but I do not understand why we don't want 8.1 here?

Reply:
Because we run it (and check whether it works) only on PHP 8.0.

Reply:
I bumped to 8.1 in #61 , green, so merged :)